theia-dart-docker/latest.package.json

```diff
@@ -0,0 +1,31 @@
+{
+    "private": true,
+    "dependencies": {
```

Comment:
Can we follow the same idea as previous images and include the following:
- enable the preference `files.enableTrash`
- include an `applicationName` (ex: Theia Dart Example)


_Example:_
https://github.com/theia-ide/theia-apps/blob/27f92587e9cee99616c5c0610aa94c96211504ae/theia-go-docker/latest.package.json#L3-L12

Reply:
sure